mmv1/templates/terraform/examples/dlp_job_trigger_bigquery_basic.tf.erb

```diff
@@ -0,0 +1,35 @@
+resource "google_data_loss_prevention_job_trigger" "<%= ctx[:primary_resource_id] %>" {
+	parent = "projects/<%= ctx[:test_env_vars]['project'] %>"
+	description = "Description"
+	display_name = "Displayname"
+
+	triggers {
+		schedule {
+			recurrence_period_duration = "86400s"
+		}
+	}
+
+	inspect_job {
+		inspect_template_name = "fake"
+		actions {
+			save_findings {
+				output_config {
+					table {
+						project_id = "project"
+						dataset_id = "dataset"
+					}
+				}
+			}
+		}
+		storage_config {
+			big_query_options {
+				sample_method = "SAMPLE_METHOD_UNSPECIFIED"
```

Comment:
Consider also adding `rows_limit` and `rows_limit_percent` to this test

Reply:
This test was intended to show that you could omit `rows_limit` and `rows_limit_percent` if the sample method was set to `SAMPLE_METHOD_UNSPECIFIED` and not defaulted to `TOP` (which seems to force one of the rows limit fields to be specified)